src/app/portlets/dot-starter/dot-starter.component.scss

```diff
@@ -0,0 +1,194 @@
+@use "variables" as *;
+
+$number-link-icon-size: 32px;
+
+:host {
+    display: block;
+    height: 100%;
+    overflow: auto;
+
+    hr {
+        border-top: 1px solid $gray-lighter;
+    }
+
+    a {
+        text-decoration: none;
+    }
+
+    .dot-starter-title {
+        color: $black;
+        font-size: $font-size-x-large;
+        margin: $spacing-5 $spacing-3 $spacing-1;
+    }
+
+    .dot-starter-description {
+        color: $gray;
+        font-size: $font-size-large;
+        margin: 0 $spacing-3 $spacing-5 $spacing-3;
+    }
+
+    .dot-starter-top-content,
+    .dot-starter-bottom-content {
+        display: flex;
+        margin: 0 auto;
+        max-width: 1084px;
+        flex-grow: 1;
+    }
+
+    .dot-starter-top-main__section {
+        background-color: $white;
+        box-shadow: $md-shadow-4;
+        flex-grow: 1;
+        margin-left: $spacing-1;
+        margin-bottom: $spacing-4;
+        margin-right: $spacing-1;
+    }
+
+    .dot-starter-top-main__block {
+        border-bottom: 1px solid $gray-lighter;
+        margin: 0;
+        min-height: 100px;
+        transition: background-color $basic-speed ease-in;
+
+        &:hover {
+            background-color: $bg-hover;
+        }
+    }
+
+    .dot-starter-top-main__link-number {
+        align-self: center;
+        text-align: center;
+        width: 71px;
```

Comment:
Every time you have a `px` you make less responsive, use rem, but check if that width is need it.